stream/crowdstrike-endpoint-security/src/crowdstrike_connector/__init__.py

```diff
@@ -1,3 +1,3 @@
-from .crowdstrike import CrowdstrikeConnector
-
-__all__ = ["CrowdstrikeConnector"]
+__all__ = ["CrowdstrikeConnector", "ConnectorSettings"]
+from crowdstrike_connector.connector import CrowdstrikeConnector
+from crowdstrike_connector.settings import ConnectorSettings
```

Comment:
Same as previous comment on another connector:

According to PEP 8 (Python's official style guide), the recommended order is module docstring, imports and module-level declarations (including __all__):

```suggestion
from crowdstrike_connector.connector import CrowdstrikeConnector
from crowdstrike_connector.settings import ConnectorSettings

__all__ = ["CrowdstrikeConnector", "ConnectorSettings"]
```

The code works but suggested changes ensure:

- Following PEP 8 and maintain consistency
- Placing imports at the top makes dependencies immediately visible when someone opens the file